fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -6811,5 +6820,67 @@ private void setReplicaStatusInternal(long tabletId, long backendId, ReplicaStat
             table.writeUnlock();
         }
     }
+
+    public void eraseDatabase(long dbId, boolean needEditLog) {
+        // remove jobs
+        Catalog.getCurrentCatalog().getLoadInstance().removeDbLoadJob(dbId);
```

Comment:
I think we don't need to call these `removeXXXJobs` functions. Because these jobs will be cancelled automatically when it find that database does not exist anymore.

Reply:
we need to remove these info from map? I don't find any other way to remove these infos.

Reply:
It's ok to leave these code in this PR. I will check these logic and refactor them later.